### PR TITLE
Added documentation for configuring session timeout

### PR DESCRIPTION
**Pull request by dsuren1**
## Description
This PR updates documentation about the configuration properties used to set up session timeout and token refresh frequency on client

**Please check if the PR fulfills these requirements**
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Documentation

## Issue
https://github.com/geosolutions-it/support/issues/3055

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: docs/developer-guide/integrations/auth.md [hunk 1]
```diff
@@ -6,6 +6,36 @@ In this section you can see the implementation details about the login / logout
 
 <img src="../img/standard-mapstore-login.png" class="ms-docimage"  style="max-width: 400px"/>
 
+### Configure session timeout
+
+The following configuration is needed to configure session timeout when using standard login (i.e not Google OAuth, Keycloak, LDAP)
+In `mapstore-ovr.properties`, add the following
+
+``` xml
```
Comment (offtherailz) [inline docs/developer-guide/integrations/auth.md:14]:
```suggestion
``` properties
```

File: docs/developer-guide/integrations/auth.md [hunk 2]
```diff
@@ -6,6 +6,36 @@ In this section you can see the implementation details about the login / logout
 
 <img src="../img/standard-mapstore-login.png" class="ms-docimage"  style="max-width: 400px"/>
 
+### Configure session timeout
+
+The following configuration is needed to configure session timeout when using standard login (i.e not Google OAuth, Keycloak, LDAP)
+In `mapstore-ovr.properties`, add the following
```
Comment (offtherailz) [inline docs/developer-guide/integrations/auth.md:12]:
```suggestion
By default MapStore session token lives 24 hours and the refresh token last forever. On application reboot anyway all the tokens are cancelled. In order to change these default. the administrator can change these defaults by adding to `mapstore-ovr.properties` file the following properties: 
```

File: docs/developer-guide/integrations/auth.md [hunk 3]
```diff
@@ -6,6 +6,36 @@ In this section you can see the implementation details about the login / logout
...
+    # Session timeout
+    restSessionService.sessionTimeout=60 #in seconds
+    restSessionService.autorefresh=false
+```
+
+- The `sessionTimeout` refers to session token expiration time (by default it’s 24 hours)
+- `autorefresh` refers to flag configured to handle automatic refresh process in the backend
+  - when set to `false`, it avoids the use of refresh token after the session token has expired, meaning, after the timeout the user will have to reconnect
+  - when set to `true`, the refresh token is used and the session extends everytime the session timeout is met
+
```
Comment (offtherailz) [inline docs/developer-guide/integrations/auth.md:24]:
```suggestion

!!! note these configurations in `mapstore.properties` are valid for the default session storage. If you are using openID or keycloak, they will not be used.

```

**Review by offtherailz — APPROVED** [review]

**Comment by tdipisa** [comment]
@ElenaGallo can we proceed for the backport of this?

**Comment by ElenaGallo** [comment]
Test passed, @dsuren1 please backport to 2023.02.xx. Thanks

**Pull request merged**